client/src/app/videos/+video-edit/video-import.component.html

```diff
@@ -0,0 +1,60 @@
+<div *ngIf="!hasImportedVideo" class="upload-video-container">
+  <div class="import-video">
+    <div class="icon icon-upload"></div>
+
+    <div class="form-group">
+      <label i18n for="targetUrl">URL</label>
+      <my-help
+        helpType="custom" i18n-customHtml
+        customHtml="You can import any URL <a href='https://rg3.github.io/youtube-dl/supportedsites.html' target='_blank' rel='noopener noreferrer'>supported by youtube-dl</a> or URL that points to a raw MP4 file. Failure to secure these rights could cause legal trouble to yourself and your instance."
```

Comment:
Ah, it's not really how I saw the message when you sent it to me.

`Failure to secure these rights could cause legal trouble to yourself and your instance.` -> `You should make sure you have diffusion rights over the content it points to, otherwise it could cause legal trouble to yourself and your instance.`

Reply:
:+1: